### PR TITLE
Amend getting started locally

### PR DESCRIPTION
**Pull request by huwd**
[Trello](https://trello.com/c/uB19t27Z/559-review-and-augment-new-starter-content-in-team-manual)

Here's a first pass at updating the local setup page

### CODE REVIEW

File: source/new-starter/prototype/index.html.md.erb [hunk 1]
```diff
@@ -7,17 +7,13 @@ review_in: 6 months
...
 # Set up the GOV.UK account manager prototype
 
-The GOV.UK account manager prototype is an application to test how users:
-
-- authenticate their data
-- authorise their data for exchange
-- stay informed of the use of their data
-- manage consent for use of their data
+The GOV.UK account manager prototype and attribute store are a central part of GOV.UK Accounts.
+You can find out more about how they work with other services in [How GOV.UK Accounts works](/new-starter/how-it-works/#how-gov-uk-accounts-works).
```
Comment (huwd) [inline source/new-starter/prototype/index.html.md.erb:12]: Should we be using relative or absolute routes?

File: source/new-starter/prototype/index.html.md.erb [hunk 2]
```diff
@@ -89,3 +89,21 @@ If you have set up the apps correctly, you will be able to access the following
 When you have set up your local account, you can [sign into your account](http://www.login.service.dev.gov.uk/) and view the manage screens.
 
 You have now set up and run the GOV.UK account manager prototype, and integrated the `finder-frontend` Brexit transition checker with the GOV.UK account manager.
+
+## Start the account manager prototype without the checker
+
+Some features do not require the checker to be present, and can be tested locally without the checker running.
+These include:
+- Signup (without any Brexit checker data) - /new-account
```
Comment (huwd) [inline source/new-starter/prototype/index.html.md.erb]: I wasn't sure what was clearer here... to just put the path or to put the full URL that you would use if you had things spun up locally?

File: source/new-starter/prototype/index.html.md.erb (hunk 1 above)
Comment (jonathanglassman) [reply source/new-starter/prototype/index.html.md.erb:12]: relative

File: source/new-starter/prototype/index.html.md.erb [hunk 3]
```diff
@@ -74,7 +70,13 @@ To alert developers when they attempt to enter a secret in the codebase, [instal
 
 ## Start the Brexit transition checker and account manager prototype apps
 
-In the command line, go to the `finder-frontend` repo folder and run `govuk-docker-up`. This starts the `finder-frontend` Brexit transition checker and its dependencies.
+To register a new GOV.UK account most users arrive through the Brexit checker.
+Without data from the Brexit checker, the user's dashboard will not show any integrations with GOV.UK services.
+
+You can spin up all apps required to follow the full user registration journey with govuk-docker.
```
Comment (jonathanglassman) [inline source/new-starter/prototype/index.html.md.erb]: A quick style point on the use of "can" (apologies if you already know this). We use "can" for optional actions. "Must" is for required actions, and "should" is for recommended actions.

Reading this, it seems that you must start these apps to follow the full user registration journey with govuk-docker. Is that correct?

File: source/new-starter/prototype/index.html.md.erb [hunk 4]
```diff
@@ -74,7 +70,13 @@ To alert developers when they attempt to enter a secret in the codebase, [instal
...
+
+In the command line, run `govuk-docker up finder-frontend-app-account`. This will start `finder-frontend` which contains the Brexit checker.
+It will also start the `govuk-account-manager-prototype` and `govuk-attribute-service-prototype` and configure it to speak to `finder-frontend` locally.
```
Comment (jonathanglassman) [inline source/new-starter/prototype/index.html.md.erb]: What is the "it" in "configure it to speak..." referring to? The `finder-frontend`, the `govuk-account-manager-prototype, or `govuk-attribute-service-prototype`? 

As a general content best practice, try to minimise the use of "it" or "they" in content to minimise ambiguity.

File: source/new-starter/prototype/index.html.md.erb (hunk 2 above)
Comment (jonathanglassman) [reply source/new-starter/prototype/index.html.md.erb]: Just to clarify as I'm slightly unclear about this. Am I right in thinking that once you've set up everything up to an including installing `detect-secrets`, you can do one of the following:

- Start the Brexit transition checker and account manager prototype apps
- Start the account manager prototype without the Brexit transition checker

And if you choose the second, then you can go directly to the URLs in this list?

If I'm understanding correctly, then I would make these descriptive links for accessibility.

File: source/new-starter/prototype/index.html.md.erb [hunk 5]
```diff
@@ -87,3 +89,21 @@ If you have set up the apps correctly, you will be able to access the following
...
+
+Some features do not require the checker to be present, and can be tested locally without the checker running.
+These include:
+- Signup (without any Brexit checker data) - /new-account
+- Login (if you have created an account in your local database) - /sign-in
+- The account dashboard (when logged in) - /account
+- The account management screen (when logged in) - /account/manage
+- The account security screen (when logged in) - /account/security
+
+To start this with govuk docker:
```
Comment (jonathanglassman) [inline source/new-starter/prototype/index.html.md.erb]: Do you need to do this part before you can do the ordered list above?

**Review by jonathanglassman — CHANGES_REQUESTED** [review]
@huwd New content looks good overall. I'm unclear about a couple of things so I've left a few comments and questions. Happy to discuss anytime.

File: source/new-starter/prototype/index.html.md.erb (hunk 3 above)
Comment (huwd) [reply source/new-starter/prototype/index.html.md.erb]: As discussed, will swap the sections.
Will amend the line to follow the GOV.UK stye guide for can/must ect.

File: source/new-starter/prototype/index.html.md.erb (hunk 2 above)
Comment (huwd) [reply source/new-starter/prototype/index.html.md.erb]: Ace, will amend those as discussed

File: source/new-starter/prototype/index.html.md.erb (hunk 4 above)
Comment (huwd) [reply source/new-starter/prototype/index.html.md.erb]: Resolved as discussed

**Review by jonathanglassman — APPROVED** [review]
@huwd content looks good. Made some last changes to bring in line with gov.uk style.

**Pull request merged**